doc/swss-schema.md

```diff
@@ -620,7 +622,27 @@ Equivalent RedisDB entry:
     12) "0"
     127.0.0.1:6379>
 
-### Configuration files
+---------------------------------------------
+### WARM\_RESTART\_TABLE
+    ;Stores application and orchdameon warm start status
+    ;Status: work in progress
+    key             = WARM_RESTART_TABLE:process_name         ; process_name is unique process identifier
+    restart_count   = 1*10DIGIT                               ; a number between 0 and 2147483647,
+                                                              ; count of warm start times.
+    state           = "init" / "restored" / "synced"
```

Comment:
can you add definition for each phase?

Reply:
restored: restore the previous
synced-> reconciled-> reconcile the previous view and new view